docs/spec_draft.md

```diff
@@ -670,6 +718,57 @@ operation.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.remainder
+
+`stablehlo.remainder(lhs, rhs) -> result`
+
+### Semantics
+
+Performs element-wise remainder of dividend `lhs` and divisor `rhs` tensors and
+produces a `result` tensor. The sign of the result is taken from the dividend,
+         and the absolute value of the result is always less than the divisor's
+         absolute value. The remainder of the any division operation `x/y` is
+         calculated as `x - n*y`, where `n` is `x/y` with its fractional part
+         truncated. For floating-point element types, this is in contrast with
+         the `remainder` operation from IEEE-754 specification where `n` is an
+         integral value nearest the exact value `x/y` with ties to even. For
+         floating-point types, the corner cases are TBD. For n-bit integer,
+         division overflow (remainder by zero or remainder of $-2^{n-1}$ with
+             $-1$) produces an implementation-defined value.
```

Comment:
Also, formatting looks really wonky here and above. Is there a particular reason to not align to the left and fill the whitespace more evenly between lines?

Reply:
The automatic vim wrap (applied using a switch). Fixed it for the two ops in this PR. I will fix the others in a separate one.